tasks/go.py

```diff
@@ -21,14 +21,15 @@
 from .utils import get_build_flags
 
 # List of modules to ignore when running lint
-MODULE_WHITELIST = [
+MODULE_ALLOWLIST = [
     # Windows
     "doflare.go",
     "iostats_pdh_windows.go",
     "iostats_wmi_windows.go",
     "pdh.go",
     "pdh_amd64.go",
     "pdh_386.go",
+    "pdhformatter.go",
```

Comment:
Why is this needed?

Reply:
So I was able to work around the issue causing the warning about `misuse of unsafe.Pointer`. At first I misinterpreted as belonging to the lint output (hence the changes to this invoke task), but in fact it is from `go vet` and there isn't a good way to mark exclusions for it, `// go vet: .. ` does seem to not work. Afaict the only option as of today is to build with `!dovet`. To summarize - once I confirm I should be able to remove addition of this file to the allow list.

Reply:
Yeah, `go vet` doesn't have an exclusion mechanism sadly :( we can use the skip list if you don't find another way around it